lib/ClangImporter/ClangImporter.cpp

```diff
@@ -3266,6 +3266,28 @@ void ClangImporter::verifyAllModules() {
 #endif
 }
 
+const clang::Type *
+ClangImporter::parseClangFunctionType(StringRef typeStr) const {
+  auto &sema = Impl.getClangSema();
+  // TODO: Pass down the filename and SourceLoc for this call.
```

Comment:
Go ahead and propagate a `SourceLoc` into this function, at least; and then if it's difficult to pass that down for some reason, so be it.  But it should be easy.

Reply:
Done.